From 1d59a36a6f6e4aff60304cfd566f3bdadc2ef420 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 28 Apr 2022 19:49:08 +0300
Subject: [PATCH] [SW-82716] gaudi3: call timestamp init routine

As part or ROM recovery init routine, this patch call timestamp
init which will be used for I2C sync mechanism.

Change-Id: I8f22caf6eebecd88fd663aa2b61d79a0318f5049
---

diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index f837228..fc68bb6 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -209,6 +209,8 @@
 
 	device_init();
 
+	arcboot_rom_init_ts();
+
 	module_handle_cmds(cmd_ops, ARRAY_SIZE(cmd_ops));
 }
 
